src/pages/GreenitFullXp/genericFullXp.tsx

```diff
@@ -23,16 +24,33 @@ const ConfirmationFullXp = React.lazy(
     () => import("pages/GreenitFullXp/ConfirmationFullXp/ConfirmationFullXp")
 );
 
+//Todo : Refactor
 const GenericFullXp = () => {
-    const currentMenuStorage = localStorage.getItem("currentMenuGreenitFullXp");
+
+    //Initialize default state of the menu
+    const startMenu =
+        (getMenuStep() === menuFullXp[3].name)
+            ? menuFullXp[3].name
+            : menuFullXp[0].name
+
+    //Delete previous cookie from payment on confirmation
+    if (startMenu === menuFullXp[3].name) {
+        localStorage.removeItem("currentMenuGreenitFullXp")
+    }
+    const currentMenuStorage = localStorage.getItem("currentMenuGreenitFullXp") || localStorage.setItem('currentMenuGreenitFullXp', startMenu)
```

Comment:
crée un constante pour le menuFullXp[3].name
```suggestion
    const paymentMenu =  menuFullXp[3].name
```